package.json

```diff
@@ -67,7 +67,7 @@
     "markdown-it": "^13.0.2",
     "markdown-it-link-attributes": "^4.0.1",
     "md5": "^2.3.0",
-    "ninja-keys": "^1.2.2",
```

Comment:
Any particular reason to use a fork of `ninja-keys`?

Reply:
@scmmishra Yes, the `ninja-keys` don't support the close event. So @muhsin-k has created a [PR](https://github.com/ssleptsov/ninja-keys/pull/54) in the official repo, which is not merged as I guess it is not maintained. The changes in the PR are updated in our forked repo and deployed in npm by Muhsin.